internal/infra/http/server.go

```diff
@@ -66,7 +69,23 @@ func (c *Component) Start() error {
 		return fmt.Errorf("hompage initialization failed: %w", err)
 	}
 
-	http.Handle("/", homepage)
+	if c.config.OTELEnabled {
+		c.logger.Debug("Enable OTEL")
+
+		otelShutdown, err := tracing.SetupOTelSDK(ctx)
```

Comment:
You could return the TracerProvider to stop it properly instead of using shutdown functions if you want to

Reply:
As discussed, I implemented it.